arches/app/views/resource.py

```diff
@@ -637,10 +637,17 @@ def get(self, request, page=None):
             except (ValueError, TypeError) as e:
                 return HttpResponseBadRequest()
 
-        totalItems = models.EditLog.objects.all().exclude(resourceclassid=settings.SYSTEM_SETTINGS_RESOURCE_MODEL_ID).count()
-
+        totalItems = (
+            models.EditLog.objects.all()
+            .exclude(resourceclassid=settings.SYSTEM_SETTINGS_RESOURCE_MODEL_ID)
+            .exclude(note="resource creation")
+            .count()
+        )
         edits = (
-            models.EditLog.objects.all().exclude(resourceclassid=settings.SYSTEM_SETTINGS_RESOURCE_MODEL_ID).order_by("timestamp")[st:end]
+            models.EditLog.objects.all()
+            .exclude(resourceclassid=settings.SYSTEM_SETTINGS_RESOURCE_MODEL_ID)
+            .exclude(note="resource creation")
+            .order_by("timestamp")[st:end]
         )
```

Comment:
Hey @njkim :wave:

Wondering if you considered consolidating this into a single query?

Reply:
My bad -- didn't notice the slice at the end. Resolving.